### PR TITLE
AppImage Automatic Updater

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #318 ([AppImage] Extend Linux Automatic Updating Section)

Adding documentation about the AppImage automatic updater.

Backport to 4.0 and 4.1

@fmoc fyi, as discussed

### CODE REVIEW

File: modules/ROOT/pages/automatic_updater.adoc [hunk 1]
```diff
@@ -24,7 +24,14 @@ If a new update is available, the ownCloud Desktop App initializes a pop-up dial
 
 === Linux
 
-Linux distributions provide their own update tools. The ownCloud Desktop App use the Linux operating system do not perform any updates on their own. The client will inform you (menu:Settings[General > Updates]) when an update is available.
+* Linux distributions provide their own update tools. +
+The ownCloud Desktop App use the Linux operating system do not perform any updates on their own. The client will inform you (menu:Settings[General > Updates]) when an update is available.
```
Comment (EParzefall) [inline modules/ROOT/pages/automatic_updater.adoc:28]:
```suggestion
The ownCloud Desktop app on a Linux operating system does not perform any updates on its own. The client will inform you (menu:Settings[General > Updates]) when an update is available.
```

File: modules/ROOT/pages/automatic_updater.adoc [hunk 2]
```diff
@@ -24,7 +24,14 @@ If a new update is available, the ownCloud Desktop App initializes a pop-up dial
...
+
+* AppImages +
+AppImages provide an auto update feature. The check process takes place when (re)starting the AppImage. If a new AppImage is found, you get the following example screen where you can decide to install the new image or not. Updates are in-place and do not produce a new filename.
```
Comment (EParzefall) [inline modules/ROOT/pages/automatic_updater.adoc:31]:
```suggestion
AppImages provide an auto-update feature. The check process takes place when (re)starting the AppImage. If a new AppImage is found, you get the following example screen where you can decide whether to install the new image or not. Updates are in-place and do not produce a new file name.
```

**Review by EParzefall — APPROVED** [review]
language review

**Review by phil-davis — APPROVED** [review]

**Comment by saw-jan** [comment]
Doc is there https://doc.owncloud.com/desktop/next/automatic_updater.html#linux and update dialog is shown for older (3.2) appImage

**Pull request merged**